.github/workflows/package-ui-test.yaml

```diff
@@ -47,3 +47,44 @@ jobs:
 
       - name: Build `ui`
         run: yarn build:ui
+
+  deploy:
+    runs-on: ubuntu-latest
+    env:
+      NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+          token: '${{ secrets.ACCESS_TOKEN }}'
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+          cache: 'yarn'
+
+      - name: Install `foundry`
+        uses: onbjerg/foundry-toolchain@v1
+        with:
+          version: nightly
+
+      - name: Install `packages`
+        run: yarn install
+        env:
+          # Fixes issue: `ethereumjs-abi: The remote archive doesn't match the expected checksum`
+          YARN_CHECKSUM_BEHAVIOR: update
+
+      - name: Install Netlify CLI
+        run: npm intall -g netlify-cli
+
+      - name: Link with project
+        run: |
+          netlify link --id ${{ secrets.NETLIFY_SITE_ID }}
+
+      - name: Deploy with Netlify
+        if: github.ref == 'refs/heads/development'
+        run: netlify deploy --branch development --prod --build
```

Comment:
We have netlify setup to automatically trigger a build once we push to main/development, we don't need to start it manually here.


Furthermore we could make this `deploy` job dependent on the `build` job and just trigger a new deployment via the netlify cli if you want?

Reply:
Ah nevermind, will adapt and deactivate auto deployment via netlify.

Reply:
> We have netlify setup to automatically trigger a build once we push to main/development, we don't need to start it manually here.

The whole point is doing the build here, so we can use the whole forge toolchain and get rid of the `out` directory files, and build the contracts on the fly

Reply:
Basically, what I want is be able to build + deploy via GHA only :) if that makes sense

Reply:
yep yep got it. Was a bit confused. Makes sense, wasn't aware the `--build` flag causes the cli to call the build command specified in the netlify.toml.

Reply:
The build step in the toml however, would miss the command to generate the `out` folder themselves, probably `forge install` `forge compile` or so.

Reply:
Just checked to deactivate the automatic deployment on netlify. Even when set to `none` it will trigger deployment of the production branch . xD
Maybe we just need to setup a branch to production whcih we don't use. Something like a `donotuse` branch or so.

<img width="946" alt="Screenshot 2022-06-10 at 10 04 08" src="https://user-images.githubusercontent.com/839848/173020624-a8d2f39c-e88e-4593-a458-c7b2e186663b.png">